lib/utils/rename.js

```diff
@@ -1,16 +1,182 @@
 'use strict'
 var fs = require('graceful-fs')
+var path = require('path')
+var rimraf = require('rimraf')
 var SaveStack = require('./save-stack.js')
 
 module.exports = rename
 
 function rename (from, to, cb) {
   var saved = new SaveStack(rename)
```

Comment:
Yup, without this errors in rename have no stacktrace at all, because they lead back to the callback being called from Node.js internals.